torch_tvm/fuse_concat.cpp

```diff
@@ -0,0 +1,59 @@
+#include "fusion_pass.h"
+#include "operators.h"
+
+using namespace torch::jit;
+
+bool isFusableCatNode(Node* node) {
+  if (node->kind() != aten::cat)
+    return false;
+  if (!node->is_constant(attr::dim))
+    return false;
+
+  auto tensors_node = node->namedInput(attr::tensors)->node();
+  if ((tensors_node->inputs().size() + node->outputs().size()) >
```

Comment:
Can you add a comment as to what is this supposed to check? And maybe `constexpr int some_var_name = 128`?